calendar-bundle/src/EventListener/PreviewUrlConvertListener.php

```diff
@@ -41,7 +41,7 @@ public function __invoke(PreviewUrlConvertEvent $event): void
 
         $request = $event->getRequest();
 
-        if (null === $request || null === ($eventModel = $this->getEventModel($request))) {
+        if (null === ($eventModel = $this->getEventModel($request))) {
```

Comment:
```suggestion
        if (null === ($eventModel = $this->getEventModel($event->getRequest()))) {
```
We could directly inline it then I think?

Reply:
Changed in f2c019aec55bade213d1f2dc37f079a4a1a031e6.